File: LV0/코드 처리하기.js
function solution(code) {
  let mode = 0;
  let answer = "";
  for (let i = 0; i < code.length; i++) {
    if (code[i] === "1") {
      if (mode === 0) {
        mode = 1;
      } else {
        mode = 0;
      }
    }
    if (code[i] !== "1") {
      if (mode === 0 && i % 2 === 0) {
        answer += code[i];
      } else if (mode === 1 && i % 2 !== 0) {
        answer += code[i];
      }
    }
  }
  return answer === "" ? "EMPTY" : answer;
}

File: LV0/코드 처리하기.py
def solution(mode):
    mode = 0
    answer = ""
    for i in range(len(code)):
        if code[i] == "1":
            if mode == 0:
                mode = 1
            else:
                mode = 0
        if code[i] != "1":
            if mode == 0 and i % 2 == 0:
                answer += code[i]
            elif mode == 1 and i % 2 != 0:
                answer += code[i]
    return answer if answer != "" else "EMPTY"